docs/ref/databases.txt

```diff
@@ -679,13 +685,17 @@ appropriate typecasting.
 SQLite notes
 ============
 
-Django supports SQLite 3.8.3 and later.
+Django supports SQLite 3.8.3 and later. sqlite3_ is required; in most
+installations, this is part of the Python Standard Library but may require
+separate installation in certain environments (such as FreeBSD).
```

Comment:
I think this actually makes it less clear. We're muddying the water for the vast majority here. Folks who are on more esoteric distributions know what they're about.